kiele

```diff
@@ -137,6 +138,7 @@ while [[ $# -gt 0 ]]; do
         --module)     module="$2"               ; shift 2 ;;
         --definition) definition="$2"           ; shift 2 ;;
         --schedule)   schedule='`'$2'`(.KList)' ; shift 2 ;;
+        --depth)      depth=$2                  ; shift 2 ;;
```

Comment:
If we're adding the `--depth` option, we need to make sure it's handled correctly by all cases of `interpret` (you've only handled the `standalone` case), and we should also make it handle correctly for `krun` option.